src/Data/Map/Internal.purs

```diff
@@ -158,6 +155,52 @@ instance traversableWithIndexMap :: TraversableWithIndex k (Map k) where
           <*> f k2 v2
           <*> traverseWithIndex f right
 
+-- | `SemigroupMap k v` provides a `Semigroup` instance for `Map` whose definition
+-- | depends on the underlying `value` type's `Semigroup` instance.
+-- | You should only use this type when you need `Data.Map` to have
+-- | a `Semigroup` instance.
+-- |
+-- | ```purescript
+-- | let
+-- |   s :: forall key value. key -> value -> SemigroupMap key value
+-- |   s k v = Unbiased.SemigroupMap (Data.Map.singleton k v)
```

Comment:
I think we can use SemigroupMap and singleton unqualified here.

Reply:
Yup! Thanks!